recipe/meta.yaml

```
@@ -28,7 +28,7 @@ requirements:
     - pytorch >=1.13.0
     - pyyaml
     - safetensors
-    - transformers >=4.18.0
+    - transformers
```

Even though `transformers==4.37.1` seems to be installed now on the CI at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=866830&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=1066, probably best to set minimum pin here to prevent https://github.com/conda-forge/peft-feedstock/pull/11#issuecomment-1853053228 from happening again.
```suggestion
    - transformers >=4.18.0
```